src/puya/ir/builder/main.py

```diff
@@ -165,6 +165,98 @@ def visit_assignment_expression(self, expr: awst_nodes.AssignmentExpression) ->
         else:
             return ValueTuple(expr.source_location, list(result))
 
+    def visit_biguint_postfix_unary_operation(
```

Comment:
There's a lot of repeated code here but it's different enough that it might not be worth refactoring

Reply:
Agreed, not worth it